pkgs/applications/networking/tcpkali/default.nix

```diff
@@ -0,0 +1,19 @@
+{stdenv, autoconf, automake, fetchurl, libtool, bison}:
+
+let version = "0.9"; in
+
+stdenv.mkDerivation {
+  name = "tcpkali-${version}";
+  src = fetchurl {
+    url = "https://github.com/machinezone/tcpkali/archive/v${version}.tar.gz";
```

Comment:
It is preferrable to use `fetchFromGitHub` when using auto-generated tarballs, as github sometimes changes compression &c, making the hash unstable.